source/common/upstream/upstream_impl.cc

```diff
@@ -1148,13 +1148,17 @@ void ClusterImplBase::onPreInitComplete() {
   }
   initialization_started_ = true;
 
-  ENVOY_LOG(debug, "initializing {} cluster {} completed",
+  ENVOY_LOG(debug, "pre-initializing {} cluster {} completed",
```

Comment:
I found this strange because it says initialization is completed, but initialization isn't actually completed until later